smoketest/__init__.py

```diff
@@ -158,19 +159,16 @@ def main():
             ))
             sys.__stdout__.flush()
             stop_event.set()
-            while filter(methodcaller('is_alive'), threads):
-                map(methodcaller('join', 0.1), threads)
+            while [t for t in threads if t.is_alive()]:
```

Comment:
Since this is repeated from above, maybe a simple method like:

```python
def alive_threads(threads):
    return (t for t in threads if t.is_alive())
```

and perhaps the callers be:

```python
while any(alive_threads(threads)):
```

Reply:
Done in aef0fb9.